### PR TITLE
[ML-5916] Cross build graphframes for multiple spark versions.

### PR DESCRIPTION
**Pull request by MrBago**
Add spark version argument to release script & allow the release script to build for multiple spark versions.

### CODE REVIEW

File: dev/release.py [hunk 1]
```diff
@@ -38,7 +38,10 @@ def verify(prompt, interactive):
               help="Push current branch and docs to this git remote.")
 @click.option("--publish-docs", type=bool, default=PUBLISH_DOCS_DEFAULT, show_default=True,
               help="Publish docs to github-pages.")
-def main(release_version, next_version, publish_to, no_prompt, git_remote, publish_docs):
+@click.option("--spark-version", multiple=True, show_default=True,
+              default=["2.3.0", "2.4.0"])
+def main(release_version, next_version, publish_to, no_prompt, git_remote, publish_docs,
```
Comment (felixcheung) [inline dev/release.py]: I'd add a blank line above this for readability

**Comment by yogeshg** [comment]
I checked that this works. +1 on the readability comment. LGTM pending that.

**Comment by mengxr** [comment]
@MrBago Could you address @felixcheung 's comment? Otherwise LGTM.

**Comment by felixcheung** [comment]
oh yes

**Pull request merged**